### ISSUE TITLE
Accessibility: Put Status Line Information In a Footer Tag

### ISSUE THREAD

**Issue by jhomme** [microsoft/vscode#98376]
Issue Type: <b>Feature Request</b>

Doing this will automatically create a landmark that screen readers can use to easily find status information. 

VS Code version: Code - Insiders 1.46.0-insider (da98a19a5d6724367bd99c8ec3b8f55054be37d1, 2020-05-20T22:21:02.272Z)
OS version: Windows_NT x64 10.0.18362

**Comment by isidorn** [comment]
Status line information is already captured by the role `status`. Thus your screen reader should read when you tell it to read status bar inforamtion. Though note there is an issue on the nvda side 
https://github.com/nvaccess/nvda/issues/11064

Can you elaborate what you mean by the "Footer Tag"

**Comment by jhomme** [comment]
Hi,

<footer>

Status info goes here.

</footer>

 

==========

Jim Homme

Skype: jim.homme

FreeChess: jhomme

Twitter: jimhomme <https://twitter.com/jimhome> 

Facebook: jimhomme

Website: jimhomme.com <https://www.jimhomme.com/>

**Comment by isidorn** [comment]
Yes, but what would be gain by it.
@joanmarie does Orca handle `footer` tags in any special way?

**Comment by jhomme** [comment]
Hi,

It would create a landmark. Screen readers have special functions for jumping among landmarks. Right now the only landmark is main. The screen reader user could go into Browse mode, press one or two keys, and they’d be at the top of the status information, and review it.

 

==========

Jim Homme

Skype: jim.homme

FreeChess: jhomme

Twitter: jimhomme <https://twitter.com/jimhome> 

Facebook: jimhomme

Website: jimhomme.com <https://www.jimhomme.com/>

**Comment by isidorn** [comment]
Assigning to June so we can experiment with this then.

**Comment by isidorn** [comment]
Just tried changing our Status Bar Part to not be `div` but a `footer`and seems to work just fine.
Alternatives are: leave it as it is, or introduce two footers, one for left side and one for right side. Or to introduce a new elemetn between the part and the left and right which would be the footer containing them

@bpasero toughts if we change this?

**Comment by bpasero** [comment]
My understanding is that `footer` is just a `div` so it is probably fine. However my understanding was always that you do not have to use the semantic HTML elements and can do everything through ARIA roles and `div` too.

**Comment by isidorn** [comment]
Exactly. We already set the appropriate ARIA roles, but it seems there would be additional benefits as @jhomme mentions. Though let's wait for more community feedback.
And yeah `footer` is just a `div`

**Comment by isidorn** [comment]
@jhomme based on your comment adding verfied label
